Developer tools/Useful software.md

```diff
@@ -0,0 +1,23 @@
+In the following list, we'll give some useful software and links that will speed up your development.
+
+[SVG 2 WP](https://svg-2-wp.goranalkovic.com/) - a super useful site created by Goran Alković, that will transform your SVG in a format compatible with JS use (either in manifest or JS files)
+
+[Postman](https://www.postman.com/) - Postman is an indispensable tool when testing any kind of API endpoints. It offers the ability to create your own personal collections, environments, testing REST API, GraphQL endpoints, and more
+
+[Diffchecker](https://www.diffchecker.com/) - Useful tool to check large amounts of diffs (PRs, texts, etc.)
+
+[Flycut](https://apps.apple.com/us/app/flycut-clipboard-manager/id442160987?mt=12) - Simple clipboard manager for developers
+
+[DevUtils](https://devutils.app/) - Powerful developer tools for your everyday tasks.
+
+[Rectangle](https://rectangleapp.com/) - Move and resize windows in macOS using keyboard shortcuts or snap areas
+
+[Sequel Ace](https://apps.apple.com/us/app/sequel-ace/id1518036000?mt=12) - Sequel Ace is an unofficial "sequel" to longtime macOS tool Sequel Pro
+
+[Table Plus](https://tableplus.com/) - Modern, native, and friendly GUI tool for relational databases: MySQL, PostgreSQL, SQLite & more
+
+[Notion](https://www.notion.so/) - Super useful tool to keep all your notes
+
+[Hover inspector for Chrome](https://chrome.google.com/webstore/detail/hover-inspector-like-in-z/jbhnejndccdjigppalbbiphlbaaehpln/related?hl=en) - Inspector is the tool which mimic display dimension between selected and hover element. It would be good for designers, QA

```

Comment:
...**mimics display dimension(s) between selected and hovered element**

..**It can be useful for designers and QA**